homeassistant/components/jvc_projector/coordinator.py

```diff
@@ -49,14 +44,10 @@ async def _async_update_data(self) -> dict[str, str]:
         except JvcProjectorAuthError as err:
             raise ConfigEntryAuthFailed("Password authentication failed") from err
 
-        old_interval = self.update_interval
-
-        if state[const.POWER] != const.STANDBY:
-            self.update_interval = INTERVAL_FAST
-        else:
-            self.update_interval = INTERVAL_SLOW
-
-        if self.update_interval != old_interval:
-            _LOGGER.debug("Changed update interval to %s", self.update_interval)
+        _LOGGER.debug(
+            "JVC Projector - get_state = power %s - input %s",
+            state["power"],
+            state["input"],
+        )
```

Comment:
Can we also revert these changes. These change logic and have nothing to do with the binary sensor platform

Reply:
@joostlek I can remove it but is a debug info that reports the projector status, at pooling time. To identify if there is any issue with communication that can invalidate the Binary Sensor status (i.e. in the last firmware update it has been added a power new status, without debug it would have been impossible to detect and diagnose)